### PR TITLE
Remove parameter from `current_time_bounded`

### PR DESCRIPTION
**Pull request by michaelmcinerney**
Most of the diff is entirely uninteresting - you can probably just ignore `DetSchedSchedule_AI.thy`. Apart from removing the various numbers, several lemmas which are now unnecessary have also been removed

### CODE REVIEW

File: lib/Simulation.thy [hunk 1]
```diff
@@ -96,7 +96,7 @@ where
 
 text \<open>
   Refinement is a global property over all executions and/or all
-  hoare triples. As this is hard to show, we define the weaker concept
+  hoare triples. As this is hard to show, we define the stronger concept
```
Comment (lsf37) [inline lib/Simulation.thy:99]: I think it turns out that they are actually equivalent, so probably better to just remove the word entirely (just "we define the concept of")

File: lib/Simulation.thy [hunk 2]
```diff
@@ -51,7 +51,7 @@ definition
 
 text \<open>
   The sequence of operations in the data type is then executed
-  in an initial state by initialising the private state, executing
+  in an initial state by initialising the observable state, executing
```
Comment (lsf37) [inline lib/Simulation.thy:54]: It has to do both. Initially we only had trivial observable state, so the private state was the important bit, but I would vote for just saying "initialising the state".

**Review by lsf37 — APPROVED** [review]
Cool, thanks for doing that, it's much nicer without the concrete numbers.

Looks like the PR needs a rebase before it can be merged.

**Review by corlewis — APPROVED** [review]
This looks great!

**Pull request merged**